examples/ts-without-babel/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const ReactRefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+module.exports = {
+  mode: isDevelopment ? 'development' : 'production',
+  entry: './src/index.tsx',
+
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        include: path.join(__dirname, 'src'),
```

Comment:
Probably use exclude `node_modules` instead of including `src`?

Reply:
Why? If our typescript only in src folder it's better. And the [doc] (https://webpack.js.org/configuration/) says _Try to avoid exclude and prefer include_.

Reply:
Argh yea. Lets keep this.